spec/components/log_summary_component_spec.rb

```diff
@@ -19,7 +19,7 @@
       expect(result).to have_text("Created 8 February 2022")
       expect(result).to have_text("by Danny Rojas")
       expect(result).to have_content("Owned by\n              DLUHC")
-      expect(result).to have_content("Managed by\n              DLUHC")
+      expect(result).not_to have_content("Managed by\n              DLUHC")
```

Comment:
Would the following be better? It would be good to get a test failure if you see the 'managed by' question pop up with _any_ organisation, rather than just failing for DLUHC only.
```suggestion
      expect(result).not_to have_content("Managed by")
```